Packs/ServiceNow/ReleaseNotes/2_5_7.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+##### ServiceNow v2
+- Added a Mirror In Closing Option in ServiceNow.
+- Updated the Docker image to: *demisto/python3:3.10.9.45313*.
+
+#### Layouts
+##### ServiceNow Ticket
+- Updated the ServiceNow Ticket Layout.
+
+#### Mappers
+##### ServiceNow - Incoming Mapper
+- Updated the ServiceNow Incoming Mapper.
```

Comment:
Please specify the field/s you've added

Reply:
Due to the PR split, these modifications have been undone, making them irrelevant for these release notes.